geode-management/src/main/java/org/apache/geode/management/runtime/PdxInfo.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.runtime;
+
+import org.apache.geode.annotations.Experimental;
+
+@Experimental
+public class PdxInfo extends RuntimeInfo {
```

Comment:
I think PdxInfo should be in sync with the configuration Pdx class.
So they should both name "diskStore"/"diskStoreName" the same.
PdxInfo should not have a "persistent" field.
PdxInfo should have "autoSerializer" and "pdxSerializer" instead of just "serializerClassName".
I think this is important for both the java apis and rest.

Reply:
Agree that they should have consistent names. But for persistent field, because there could be a case when a server can have a diskstore, but not persistent, in reality, we should report that case on a member. This won't happen in configuration, but could happen on a server

Reply:
When that happens (i.e. persistent is false and diskStore != null) then the product ignores the diskStore. So if we said the PdxInfo will show the diskStore used by pdx then in this case it would be correct to return null since pdx is not using a disk store. I think this is reasonable and allows us not to expose the old way of configuring it with two attributes.

Reply:
Ok, updated the realizer to include this logic